scipy/signal/_filter_design.py

```diff
@@ -2565,7 +2565,7 @@ def iirfilter(N, Wn, rp=None, rs=None, btype='band', analog=False,
     if fs is not None:
         if analog:
             raise ValueError("fs cannot be specified for an analog filter")
-        Wn = 2*Wn/fs
+        Wn = Wn / (fs/2)
```

Comment:
Is there a reason not to do this inplace?
```suggestion
        Wn /= (fs/2)
```

Reply:
Yes on my machine it gives an arror if i do it in place
`Cannot cast ufunc 'divide' output from dtype('float64') to dtype('int32') with casting rule 'same_kind'`